docker-compose.yml

```diff
@@ -1,34 +1,34 @@
 version: '3'
 services:
+  # dev node with runtime v37 and client 1.2.0
   testnet_node_alice:
     container_name: testnet_node_alice
     image: cennznet/cennznet:1.2.0
     command:
-      - --chain=dev
-      - --base-path=/mnt/node
-      - --alice
-      - --validator
+      - --dev
+      - --tmp
+      - --no-mdns
       - --unsafe-ws-external
       - --unsafe-rpc-external
-      - --rpc-cors=all
     ports:
       - '9933:9933'
       - '9944:9944'
-      - '30333:30333'
-    networks:
-      - api_testnet
-
+  # dev node with runtime v36 and client 1.1.1
+  v36_node:
+    container_name: v36_node
+    image: cennznet/cennznet:1.1.1
+    command:
+      - --dev
+      - --no-mdns
+      - --unsafe-ws-external
+    ports:
+      - '9945:9944'
```

Comment:
@KarishmaBothara Why is this 9945, now that you have changed the v36 node port?

Reply:
I haven't touched this file

Reply:
I know, but I guess @holygits has set it 9945 so that when the dockers running locally, we can distinguish which is running v36

Reply:
this one is taken care of in config endpoints